core/src/components/chip/chip.scss

```diff
@@ -8,24 +8,23 @@
   --background: #{rgba($text-color-rgb, .12)};
   --color: #{rgba($text-color-rgb, .87)};
 
-  @include border-radius(16px);
+  @include border-radius(calc(16 / 14 * 1em));
```

Comment:
With this code, the corners aren't fully rounded if the text is wrapping.

Should we instead do e.g. a hack of 9999px to make sure all four corners round equally and the short corners are fully rounded? I didn't see anywhere in the code that we're currently doing that, so I wasn't sure what we want to do here.

with current code:
<img width="269" alt="Screenshot 2023-07-25 at 4 15 18 PM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/2edc167a-b199-43c5-97e0-d4b38c16fb51">

with 9999px:
<img width="270" alt="Screenshot 2023-07-25 at 4 10 38 PM" src="https://github.com/ionic-team/ionic-framework/assets/14926794/f4ad616e-69ec-4c46-ae5c-d48c73ad11b2">